lib/sles4sap_publiccloud.pm

```diff
@@ -306,7 +311,8 @@ sub check_takeover {
 sub enable_replication {
     my ($self) = @_;
     my $hostname = $self->{my_instance}->{instance_id};
-    die("Fenced database '$hostname' is not offline") if ($self->is_hana_online());
+    die("Fenced database '$hostname' is not offline") if ($self->is_hana_database_offline());
```

Comment:
Before enabling replication the fenced database should be offline. Is the logic correct here?